addon/components/materialize-button.js

```diff
@@ -21,13 +12,25 @@ export default Ember.Component.extend({
   buttonType: null,
   isFlat: Ember.computed.equal('buttonType', 'flat'),
   isDisabled: false,
-  buttonClass: function(){
+
+  didInsertElement: function(){
+    this._super(...arguments);
+    Ember.run.scheduleOnce('afterRender', this, function(){
+      var Waves = window.Waves || {};
+      if(typeof Waves.displayEffect === 'function'){
+        Waves.displayEffect();
+      }
+    });
+  },
+
+  buttonClass: Ember.computed('buttonType', function(){
     if(!this.get('buttonType')){
       return 'btn';
     } else {
       return 'btn-' + this.get('buttonType');
```

Comment:
note to self: `Ember.get` should be called only once, and can be expressed as a ternary operator w/ template strings